cypress/e2e/smoke/landing.cy.js

```diff
@@ -0,0 +1,35 @@
+describe('Landing page', () => {
+  it('redirects to welcome page', () => {
+    cy.visit('/')
+
+    cy.url().should('include', '/welcome')
+  })
+
+  it('should require accept "Updates" cookies to display Beamer', () => {
+    cy.visit('/')
+
+    // Way to select the cookies banner without an id
+    cy.contains('We use cookies to provide').parent('div').should('be.visible')
+    cy.contains('We use cookies to provide').parent('div').contains('Accept selection').click()
```

Comment:
Similar here, could we do `cy.contains('Accept selection').click()` like we do with other buttons/links?